tests/conftest.py

```diff
@@ -337,3 +338,49 @@ def monkeypatch_session():
     m = MonkeyPatch()
     yield m
     m.undo()
+
+
+# Standardized dummy projects for testing version-search
+# behavior under different stable/prerelease circumstances.
+# In particular, created to support the 'latest' endpoints of
+# https://github.com/pypa/warehouse/pull/8615
+
+ProjectData = namedtuple("ProjectData", ["project", "latest_stable", "latest_pre"])
+
+
+@pytest.fixture(scope="function")
```

Comment:
IIRC `scope="function"` is the default value, and if there's no argument, you can omit parens.
So that could be 
```suggestion
@pytest.fixture
```